metadata.json

```diff
@@ -23,7 +23,13 @@
     }
   ],
   "dependencies": [
-    { "name": "puppetlabs/stdlib", "version_requirement": ">= 3.2.0 <5.0.0" },
-    { "name": "herculesteam/augeasproviders_grub", "version_requirement": "2.x" }
+    { "name": "puppetlabs/stdlib", "version_requirement": ">= 4.13.0 <7.0.0" },
```

Comment:
Current latest version of stdlib is 5.2.0 so `<7.0.0` should probably be `<6.0.0`